packages/@glimmer/interfaces/lib/runtime/arguments.d.ts

```diff
@@ -58,5 +58,6 @@ export interface CapturedNamedArguments extends Record<string, Reference> {
 
 export interface Arguments {
   positional: readonly unknown[];
+  errors: Record<string, Error>;
```

Comment:
Where all is this interface used? If it's public, i'm worried about adding to it.

If it's internal, than it's fine

Reply:
it would be used for the inspector, to distinguish if an arg throws an error, or if it return an Error.
as it could also just be a `return new Error()`